config/install/entity_usage.settings.yml

```diff
@@ -0,0 +1,5 @@
+track_enabled_base_fields: false
```

Comment:
> If enabled, relationships generated through non-configurable fields (basefields) will also be tracked.

I'm not grokking this at the moment.

Reply:
/contrib/entity_usage/src/EntityUsageTrackBase.php 

`getReferencingFields` method

Reply:
I think thats if a base field such as `title` or `changed` have a entity reference, it wont track those. i see no circumstance we'd ever want/need that

Reply:
yeah, that is my foggy understanding of this as well. Default is false and I have been happy with the usage reports so far.